scheduler/src/cook/mesos/mesos_compute_cluster.clj

```diff
@@ -19,6 +19,7 @@
             [clojure.tools.logging :as log]
             [cook.compute-cluster :as cc]
             [cook.config :as config]
+            [cook.kubernetes.metrics :as metrics]
```

Comment:
Let's not use the `cook.kubernetes.metrics` function from the Mesos compute cluster.